OpenRA.Mods.Common/Widgets/Logic/GlobalChatLogic.cs

```diff
@@ -38,9 +38,8 @@ public GlobalChatLogic(Widget widget)
 			inputBox.IsDisabled = () => Game.GlobalChat.ConnectionStatus != ChatConnectionStatus.Joined;
 			inputBox.OnEnterKey = EnterPressed;
 
-			// Set a random default nick
-			if (Game.Settings.Chat.Nickname == new ChatSettings().Nickname)
-				Game.Settings.Chat.Nickname += Game.CosmeticRandom.Next(100, 999);
+			if (string.IsNullOrEmpty(Game.Settings.Chat.Nickname))
```

Comment:
I don't think this really helps: this will only ever run once (until the player clears their settings), and at this point the player name is probably still "Newbie".

If we wanted to address #9728 properly (i'm not sure that we want to) we would need to remove `Game.Settings.Chat.Nickname` completely and populate the `nicknameBox.Text` with the player name always.